src/client.ts

```diff
@@ -222,6 +243,19 @@ export class Client {
     return this.executeRequestOperation(operation);
   };
 
+  mutation({
+    query,
+    variables,
+    context,
+  }: PromisfiedOperation): Promise<OperationResult> {
+    const request = createRequest(query, variables);
+    return pipe(
+      this.executeMutation(request, context),
+      take(1),
+      toPromise
```

Comment:
So for the reason of multiple results I'd still opt for no promisification by default. It doesn't make sense to create a shortcut API that then compromises on only using promises which also hides some of what you could otherwise do without problems 😅

Reply:
So you would remove take(1) and toPromise and just give the user the option to subscribe?

Reply:
Yep, to subscribe, call toPromise, themselves, etc. Otherwise I wouldn't even be sure what we'd be adding these methods for 😅

Admittedly promises are widespread, but are we really just adding these shortcuts for server code?

Because in any other case the cancellation and stream will likely be important to people

Reply:
What do you think about this: we offer the possibility for a fourth argument that is called `promisify` this will return a `take(1), toPromise`. As a comfort API or are we just settling on `return this.executeMutation(request, context)`